provision.sh

```diff
@@ -38,7 +38,7 @@ echo "---------------------------------------------"
 # And that would prevent ~/.bashrc to work properly because \r would be unrecognized
 # Ruby needed for heroku toolbelt
 # notice that this is not a rigorous Ruby install, where we typically use rvm
-apt-get install -y --no-install-recommends heroku-toolbelt build-essential dos2unix python-pip man ruby python-dev python3-dev libpq-dev postgresql postgresql-contrib 
+apt-get install -y --no-install-recommends heroku-toolbelt build-essential dos2unix python-pip man ruby python-dev python3-dev libpq-dev postgresql postgresql-contrib curl
```

Comment:
@koustuvsinha The `libpython2.7-dev` package takes up 22.0 MB in download, but we don't even use it because we are using Python 3 with postgreSQL. Should we remove it?